### ISSUE TITLE
VPP/Modbus

### ISSUE THREAD

**Issue by Feierprinz** [Lu-Fi/GrowattSPH-FHEM-Modbus-Module#1]
Hi,

was macht denn der Modbus bei dem Script genau und wie bindet man das in FHEM ein? Laut Code liest du ja auch Daten aus, machst du das an dem VPP-Anschluss (RS485)?

LG
Swen

**Comment by Lu-Fi** [comment]
Ja, das geht über RS485. Einfach einen Adapter (USB -> RS485) per Modbus in Fhem einbinden.
https://wiki.fhem.de/wiki/Modbus

Dann die Module in den FHEM Ordner kopieren und definieren.

P.S.: Ich nutze diesen Adapter: https://www.amazon.de/gp/product/B016IG6X7I

Gruß
